.kokoro/build.sh

```diff
@@ -27,23 +27,27 @@ export PYTHONUNBUFFERED=1
 # Debug: show build environment
 env | grep KOKORO
 
-# Setup service account credentials.
-export GOOGLE_APPLICATION_CREDENTIALS=${KOKORO_GFILE_DIR}/service-account.json
+if [[ -f "${KOKORO_GFILE_DIR}/service-account.json" ]]; then
```

Comment:
Hmm, when is this file not going to exist?  Is it missing on the `docs-presubmit` job, but present for normal builds?

Reply:
On the `docs-presubmit` job it is not present.  Alternatively, we can add `RUN_SYSTEM_TESTS` false as an environment variable for the `docs-presubmit` job and apply [this change](RUN_SYSTEM_TESTS) in noxfile.py to skip system tests. I'm not up to speed on kokoro yet so I'd need some time to figure that out.

Reply:
oops, fixed a broken link.

Reply:
Why would systests be running in the `docs-presubmit` job?

```bash
$ git remote -v
origin	git@github.com:googleapis/python-ndb (fetch)
origin	git@github.com:googleapis/python-ndb (push)
$ git status
On branch master
Your branch is up to date with 'origin/master'.

nothing to commit, working tree clean
$ tail -n 5 .kokoro/docs/docs-presubmit.cfg 
# Only run this nox session.
env_vars: {
    key: "NOX_SESSION"
    value: "docs docfx"
}
```

Reply:
Hmmm, `python-ndb` seems to be missing `.trampolinerc`.

Reply:
I've added .trampolinerc